server/app/auth/oidc/CustomOidcLogoutRequest.java

```diff
@@ -0,0 +1,87 @@
+package auth.oidc;
+
+import com.google.common.base.Strings;
+import com.nimbusds.openid.connect.sdk.LogoutRequest;
+import java.net.URI;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class CustomOidcLogoutRequest extends LogoutRequest {
+  /** The optional post-logout redirection param and URI. */
```

Comment:
I believe this comment is meant to apply to both of the below variables but it only got attached to the topmost one. Likely an artifact of bin/fmt. Can you add a separate comment for the URI below?